Yoast/wordpress-seo
# Introduce creation of PHP string files

## Summary

This PR can be summarized in the following changelog entry:
n/a

## Technical details

For translate.wordpress.org we need to create PHP files for all the strings we need to have translated for the JavaScript. This way we can then download the strings from translate.wordpress.org and use them in our JavaScript.

## Test instructions

This PR can be tested by following these steps:

* Test if `grunt build:i18n` will create two PHP files that contain strings for YoastSEO.js and yoast-components.
* Check to see that these PHP files (for YoastSEO.js and yoast-components) contain all the strings necessary for translation of these.
